src/pfe/file-watcher/server/test/scripts/setup.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+CW_DIR=~/codewind
```

Comment:
I think the tests should be run against the codewind repository where the test.sh script is run. We can avoid cloning the repository again in another location and the added benefit is that If a user clones the codewind repository and makes changes to the tests they can run tests with their current changes without having to push the changes and specify a different branch.